fearless-utils/src/test/java/jp/co/soramitsu/fearless_utils/scale/dataType/EnumTest.kt

```diff
@@ -23,7 +24,7 @@ class EnumTest {
     fun `should serialize an deserialize enum`() {
         val hex = "0x01"
 
-        val parsed = EnumTypeTest.read(hex)
+        val parsed = EnumTypeTest.read(hex.fromHex())
```

Comment:
Why do we need `fromHex()`. There was an overload of `read()` that accepts string and performs `fromHex()` by itself